crates/subspace-malicious-operator/src/lib.rs

```diff
@@ -19,3 +19,282 @@
 mod malicious_bundle_producer;
 mod malicious_bundle_tamper;
 pub mod malicious_domain_instance_starter;
+
+use clap::Parser;
+use sc_cli::{
+    generate_node_name, ChainSpec, CliConfiguration, Role, RunCmd as SubstrateRunCmd, RunCmd,
+    SubstrateCli,
+};
+use sc_service::config::{KeystoreConfig, NetworkConfiguration};
+use sc_service::{BasePath, BlocksPruning, Configuration, DatabaseSource};
+use sc_subspace_chain_specs::ConsensusChainSpec;
+use serde_json::Value;
+use sp_domains::DomainId;
+use std::num::ParseIntError;
+use subspace_node::chain_spec;
+use subspace_node::domain::evm_chain_spec;
+
+fn parse_domain_id(s: &str) -> Result<DomainId, ParseIntError> {
```

Comment:
Not related to this PR, but would be cleaner to have `impl FromStr for DomainId`